packages/x-date-pickers/src/DateTimeField/DateTimeField.tsx

```diff
@@ -3,15 +3,19 @@ import PropTypes from 'prop-types';
 import MuiTextField from '@mui/material/TextField';
 import { useThemeProps } from '@mui/material/styles';
 import { useSlotProps } from '@mui/base/utils';
-import { refType } from '@mui/utils';
```

Comment:
Was there some reason why we removed https://github.com/mui/mui-x/pull/10273 solution in several particular cases? They seem like regressions. 🙈